src/Resources/public/js/select2/i18n/en.js

```diff
@@ -0,0 +1,3 @@
+function noResultsCreate(url_action, select_id, field_name) {
```

Comment:
Do we really need to suggest the creation when no existing result is found ? It requires many translations files whose maintenance is annoying...
Why not displaying a help message in the HTML of autocomplete (Twig level) and benefit of YAML translation files (that already exist) ?